x-pack/plugins/secops/public/pages/hosts/host_details.tsx

```diff
@@ -144,6 +146,31 @@ const HostDetailsComponent = pure<HostDetailsComponentProps>(
                           />
                         )}
                       </UncommonProcessesQuery>
+                      <EventsQuery
+                        endDate={to}
+                        filterQuery={getFilterQuery(hostId, filterQueryExpression, indexPattern)}
+                        poll={poll}
+                        sourceId="default"
+                        startDate={from}
+                        type={hostsModel.HostsType.page}
+                      >
+                        {({ events, loading, id, refetch, totalCount, pageInfo, loadMore }) => (
+                          <EventsTableManage
+                            id={id}
+                            refetch={refetch}
+                            setQuery={setQuery}
+                            data={events!}
+                            loading={loading}
+                            startDate={from}
+                            totalCount={totalCount}
+                            nextCursor={getOr(null, 'endCursor.value', pageInfo)!}
+                            tiebreaker={getOr(null, 'endCursor.tiebreaker', pageInfo)!}
+                            hasNextPage={getOr(false, 'hasNextPage', pageInfo)!}
+                            loadMore={loadMore}
+                            type={hostsModel.HostsType.page}
```

Comment:
You'll want `type={hostsModel.HostsType.details}` here otherwise it'll share the same redux data (and thus row display settings) of the table on the Hosts page.

Reply:
Updated with a single type at the top to make it easier to review this file for places that aren't the same type so we don't accidentally end up with mixed pages.